modules/metastore/src/Form/DkanDataSettingsForm.php

```diff
@@ -89,7 +89,7 @@ public function buildForm(array $form, FormStateInterface $form_state) {
 
     $form['html_allowed_properties'] = [
       '#type' => 'checkboxes',
-      '#title' => $this->t('Properties that allow HTML'),
+      '#title' => $this->t('Dataset properties that allow HTML'),
```

Comment:
Do we consider distribution properties other referenced entity properties also dataset properties? That was the reason I used just "properties"

Reply:
FWIW I think this is fine, on this page all properties are presented as

Reply:
Why are we listing every property, most of these should never have html allowed.

Reply:
We actually aren't listing every property - only string properties. But DKAN prides itself on swappable schemas, so we cannot hardcode assumptions about what the properties are.

Reply:
Not suggesting we hardcode anything, just be more specific where html could be useful. Can this config be put into the *.ui.json files?

Reply:
@janette that's an interesting idea. The thing is this would apply for harvests and API posts as well, so it's not exactly intuitive that such config would live in the ui schema... but in theory we could check it and allow HTML in any text area fields, or add a property to the UI schema like `allowHTML`. Hmmm....

Reply:
I would say this would be a different ticket; the one I was working stored the configuration as a Drupal configuration setting. You could re-write it to be part of the schema, but as Dan said it would need to work for harvests and API posts which don't really use the ui schema.

Reply:
Yeah. And I think "dataset" is appropriate here, we are showing the dataset and including other schemas within it but they are all really experienced as part of the dataset tree.